### ISSUE TITLE
Brainstrom on compare features

### ISSUE THREAD

**Issue by sanjay-kv** [recodehive/awesome-github-profiles#724]
### Discussed in https://github.com/recodehive/awesome-github-profiles/discussions/702

<div type='discussions-op-text'>

<sup>Originally posted by **sanjay-kv** October  9, 2024</sup>
here is the link: https://recodehive.github.io/awesome-github-profiles/pages/compare.html</div>

**Comment by anchalchaturvedi08** [comment]
@sanjay-kv can you assign this task to me, Sir?

**Comment by sanjay-kv** [comment]
there is a discussion tagged we have to brain storm there before working on this.

**Comment by anchalchaturvedi08** [comment]
Are you saying that we need to discuss our ideas before starting the work?

**Comment by anchalchaturvedi08** [comment]
I would like to understand the exact issue with the comparison features so that I can suggest possible solutions. Could you please provide more details about the problem we are facing?

**Comment by sanjay-kv** [comment]
Discussed in https://github.com/recodehive/awesome-github-profiles/discussions/702

**Comment by YooAshu** [comment]
we can compare total stars earned in all repos combined
and comparing their readmes side-by-side is also a good idea i think

**Comment by sanjay-kv** [comment]
@YooAshu thats a good one you can raise a new issue and start working, I will assign to you.

**Comment by YooAshu** [comment]
@sanjay-kv I have raised issues regarding this
